### ISSUE TITLE
Copy list-views between projects

### ISSUE THREAD

**Issue by okms** [Puzzlepart/prosjektportalen365#890]
**Is your feature request related to a problem? Please describe.**
Make it easy to copy views between projects. This may mitigate the need for some customers to create a project type if their setup is only applicable for a handfull of projects.

**Describe the solution you'd like**
Pull the solution from https://www.blimped.nl/how-to-copy-views-in-sharepoint/ (with attribution) and include in prosjektportalen webparts

**Additional context**
https://github.com/pnp/sp-dev-fx-webparts/tree/main/samples/react-copy-views

![copy-views](https://user-images.githubusercontent.com/170147/201303592-9c5e3107-55bf-4e5b-8157-f8b6f22dc007.gif)

**Comment by olemp** [comment]
> Pull the solution from https://www.blimped.nl/how-to-copy-views-in-sharepoint/ (with attribution) and include in prosjektportalen webparts

As it's not a web part I propose putting it in ProjectExtensions solution.

**Comment by Remi749** [comment]
@olemp , whats the reason for closing the issue?

**Comment by olemp** [comment]
> @olemp , whats the reason for closing the issue?

It's not planned now, and it's not a request from a customer. I don't see the gains of this feature. Feel free to reopen if we want to implement this though @pzljanb @Remi749

**Comment by Remi749** [comment]
> 

I would like to see this feature being implemented at some time in the future. If we're closing the issue it will be forgotten. Issues waiting for funding or just waiting in general shouldn't be closed is my opinion.

**Comment by olemp** [comment]
> > > @olemp , whats the reason for closing the issue?
> > 
> > 
> > It's not planned now, and it's not a request from a customer. I don't see the gains of this feature. Feel free to reopen if we want to implement this though @pzljanb @Remi749
> 
> I would like to see this feature being implemented at some time in the future. If we're closing the issue it will be forgotten. Issues waiting for funding or just waiting in general shouldn't be closed, is my opinion.

Sure, but what's the usecase for this? If several projects want the same views it would be better to add those views to the project template (JSON), no?

**Comment by Remi749** [comment]
> > > > @olemp , whats the reason for closing the issue?
> > > 
> > > 
> > > It's not planned now, and it's not a request from a customer. I don't see the gains of this feature. Feel free to reopen if we want to implement this though @pzljanb @Remi749
> > 
> > 
> > I would like to see this feature being implemented at some time in the future. If we're closing the issue it will be forgotten. Issues waiting for funding or just waiting in general shouldn't be closed, is my opinion.
> 
> Sure, but what's the usecase for this? If several projects want the same views it would be better to add those views to the project template (JSON), no?

You are correct, it should of course be done in the project template. This issue is more a "nice to have" feature. Let's say the customer has created X amount of projects and later made new views. This would make it easier for customers to just copy the views. Would like @okms input on this as well before we close it for good.

**Comment by olemp** [comment]
> > > > > @olemp , whats the reason for closing the issue?
> > > > 
> > > > 
> > > > It's not planned now, and it's not a request from a customer. I don't see the gains of this feature. Feel free to reopen if we want to implement this though @pzljanb @Remi749
> > > 
> > > 
> > > I would like to see this feature being implemented at some time in the future. If we're closing the issue it will be forgotten. Issues waiting for funding or just waiting in general shouldn't be closed, is my opinion.
> > 
> > 
> > Sure, but what's the usecase for this? If several projects want the same views it would be better to add those views to the project template (JSON), no?
> 
> You are correct, it should of course be done in the project template. This issue is more a "nice to have" feature. Let's say the customer has created X amount of projects and later made new views. This would make it easier for customers to just copy the views. Would like @okms input on this as well before we close it for good.

It was created as a suggestion from @okms. @pzljanb can make the call if this is something we should include in the solution. Like I mentioned above we should‘t include this as-is as the look and feel is quite different from the rest of our components which is why I put the https://github.com/Puzzlepart/prosjektportalen365/labels/funding%20required label.
